R/write_timeseries_dsg.R

```diff
@@ -2,21 +2,24 @@
 #'
 #'
 #'@param nc_file A string file path to the nc file to be created.
-#'@param times A vector of times. Must be of type \code{\link{POSIXct}} or an attempt to 
-#'convert it will be made using \code{\link{as.POSIXct(times)}}.
+#'@param station_names A vector of names for each station to be added to the file.
+#'@param times A vector of times. Must be of type \code{POSIXct} or an attempt to 
+#'convert it will be made using \code{as.POSIXct(times)}.
 #'@param lats Vector of latitudes 
 #'@param lons Vector of longitudes
 #'@param data \code{data.frame} with each column corresponding to a station. Rows correspond to 
 #'time steps. nrow must be the same length as times. Column names must match station names.
 #'@param alts Vector of altitudes (Optional)
-#'@param data_units Character vector of observation units. Length must be the same as number 
+#'@param data_unit Character vector of observation units. Length must be the same as number 
```

Comment:
Used data_units above (and was list instead of vector). Should these two sets be the same?